tox.ini

```diff
@@ -39,9 +39,8 @@ deps =
     coverage
 
     django-formtools
-    django-phonenumber-field>=0.7.2,<0.99
+    django-phonenumber-field>=0.7.2,<2.99
```

Comment:
Shouldn't this match what's in setup.py?

Reply:
Apologies, I had overlooked the lower bounds.